scripts/index.js

```diff
@@ -1 +1,21 @@
-console.log('coucou')
+//Need server address filled in URL
+function createNewPost(title, content) {
+    const url = "server address";
+    const date = {title, content};
```

Comment:
The format you are sending isn't correct ,please check below 
 **{
    "userId": 1,
    "id": 1,
    "title": "sunt aut facere repellat provident occaecati excepturi optio reprehenderit",
    "body": "quia et suscipit\nsuscipit recusandae consequuntur expedita et cum\nreprehenderit molestiae ut ut quas totam\nnostrum rerum est autem sunt rem eveniet architecto"
  },**